openfl/interface/interactive_api/experiment.py

```diff
@@ -284,8 +311,12 @@ def _prepare_plan(self, model_provider, task_keeper, data_loader,
                                               'kwargs': task_keeper.task_settings[name]}
             else:
                 # This is a validation type task (not altering the model state)
-                for name_prefix, apply_kwarg in zip(['localy_tuned_model_', 'aggregated_model_'],
-                                                    ['local', 'global']):
+                if self.train_task_exist:
+                    adapt_validation_task_dcit = zip(['localy_tuned_model_', 'aggregated_model_'],
+                                                     ['local', 'global'])
+                else:
+                    adapt_validation_task_dcit = zip(['aggregated_model_'], ['global'])
+                for name_prefix, apply_kwarg in adapt_validation_task_dcit:
```

Comment:
`adapt_validation_task_dcit` -> `validation_task_types`

Maybe it could be something like:
```python
validation_task_types = [('global', 'aggregated_model_')]
if train_task_exist:
    validation_task_types.insert(0, ('local', 'localy_tuned_model_'))

for task_type, task_prefix in validation_task_types:

```